README.md

```diff
@@ -70,7 +70,7 @@ These scripts can all be run from the root level of the repo:
 
 #### Theme scripts
 
-You can also use the following scripts to [build and preview a theme](/guides/SITE-PACKAGES-AND-THEMES.md):
```

Comment:
I'm a little confused by this change, why did it change from a relative URL? Is this the only example of a hardcoded link going to another part of the page?

Reply:
Thanks this should not have changed 👍

Reply:
This page got moved into the design system doc site and not only on GitHub now 👍